### DISCUSSION TITLE
Codeium extension

### DISCUSSION THREAD

**Discussion by y1100101s** [eclipse-theia/theia#13909] [General] [answered]
Hello everybody. Looks like I can't figure out how to make codeium work. It requires Sign in. But after clicking it - nothing happens. No new browser window, nothing. Is there a workaround?

**Comment by JonasHelming** [comment]
@tsmaeder

**Comment by tsmaeder** [comment]
@y1100101s please open an issue with steps to reproduce. Does this rely on github authentication?

**Comment by EpicStuff** [comment, marked as answer]
i managed to sign in by downloading the extension on vscode then pasting the authentication token I got into theia

**Reply by tsmaeder** [reply]
@EpicStuff still interested in how you run into the issue. Where do you "paste the authentication token into theia"?

**Reply by EpicStuff** [reply]
I searched codeium auth in the command palette and selected `Codeium: Provide Authentication Token` and pasted the token into the popup
